tests/integration/test_plotter.py

```diff
@@ -688,3 +690,73 @@ def test_plot_design_point(modeler: Modeler, verify_image_cache):
     pl.show(
         screenshot=Path(IMAGE_RESULTS_DIR, "test_plot_design_point.png"),
     )
+
+
+def test_plot_revolve_sketch_normal(modeler: Modeler):
+    """Test plotting of a sketch revolved around an axis."""
+    # Initialize the donut sketch design
+    design = modeler.create_design("quarter-donut")
+
+    # Donut parameters
+    path_radius = 5
+    profile_radius = 2
+
+    # Create the circular profile on the XZ-plane centered at (5, 0, 0)
```

Comment:
```suggestion
    # Create the circular profile on the XZ plane centered at (5, 0, 0)
```
This wording is probably better than the "and centered" wording. Perhaps search and replace to make consistent?